src/components/table/use-cases/traces.tsx

```diff
@@ -356,6 +358,11 @@ export default function TracesTable({
       enableHiding: true,
       enableSorting: true,
     },
+    {
+      accessorKey: "tags",
+      header: "Tags",
```

Comment:
can you add `id: "tags",` here?

Reply:
Done 🙏🏻
Should we add id's to the other columns as well? Maybe we can have a quick follow up why we need or not need an id for certain columns. 😊